source/nodejs/adaptivecards-site/themes/adaptivecards/scripts/filter-table.js

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// Used for custom rendering of the table element
+// We assume that the first header row cell contains the title of the table, wrapped in braces
+// We then remove the title from the first cell
+hexo.extend.filter.register('marked:renderer', function(renderer) {
+  renderer.table = function(header, body) {
+
+    // Example header string:
+    // <tr>
+    // <th>{ActionSet Inherited properties}Property</th>
+    // <th>Type</th>
+    // <th>Required</th>
+    // <th>Description</th>
+    // <th>Version</th>
+    // </tr>
+
+    // Index of the end of the title
+    var closingIndex = header.indexOf("}");
```

Comment:
We might want to handle more edge cases here? I'm not sure if other areas of our website will be impacted here